operator/config/samples/options_v1alpha1_keptnconfig.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: options.keptn.sh/v1alpha1
+kind: KeptnConfig
+metadata:
+  labels:
+    app.kubernetes.io/name: keptnconfig
+    app.kubernetes.io/instance: keptnconfig-sample
+    app.kubernetes.io/part-of: keptn-lifecycle-toolkit
+    app.kuberentes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: keptn-lifecycle-toolkit
+  name: keptnconfig-sample
+spec:
+  # TODO(user): Add fields here
```

Comment:
will you add the example in the next pr?

Reply:
yes, it's gonna be in the next PR